index.html

```diff
@@ -2,10 +2,43 @@
 <html>
   <head>
     <title>Relaxr</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600,800" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
     <header>
+    	<h1>Relaxr</h1>
+    	<center>
+    	<h2>Get piece of mind with a single tap</h2>
+    	<a href="#" class="button1">Get it Now</a></center>
     </header>
+   	<article class="Benefits>">
+    	<h3>Benefits</h3>
+	    	<p class="b1">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+		<ul>
+			<li>Schedule meetings for you</li>
+			<li>Excel automation to complete your work for you</li>
+			<li>Responds to emails on your behalf</li>
+			<li>Does all your work for you with our revolutionary AI technology</li>
+		</ul>
+	</article>
+		<article class="testimonial">
+			<center>
+				<i>
+				<p>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."</p>
+				</i>
+				<p class="name">- Amanda, Intuit</p>
+			</center>		
+		</article>
+		</div>
+		<article class="signup">
+			<center><a href="#" class="button2">Sign Up Now!</a></center>
+		</article>
+    <footer>
+	    <h2 class="logo">Relaxr</h2>
+	    <a href="#" <img src="images/twitter.png" class="twitter"></a>
```

Comment:
These images are not working because you are missing a closeing `>` after your `href="#"`.